src/main/kotlin/no/fdk/concept_catalog/security/SecurityConfig.kt

```diff
@@ -6,27 +6,42 @@ import org.springframework.boot.autoconfigure.security.oauth2.resource.OAuth2Res
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.http.HttpMethod
+import org.springframework.security.config.Customizer
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator
 import org.springframework.security.oauth2.jwt.*
 import org.springframework.security.web.SecurityFilterChain
+import org.springframework.security.web.header.writers.StaticHeadersWriter
 import org.springframework.security.web.util.matcher.RequestMatcher
+import org.springframework.web.cors.CorsConfiguration
 
 @Configuration
 open class SecurityConfig {
 
     @Bean
     open fun filterChain(http: HttpSecurity): SecurityFilterChain {
-        http.csrf().disable()
-            .cors().and()
+        http.csrf { csrf -> csrf.disable() }
+            .headers { headers ->
+                headers.addHeaderWriter(
+                    StaticHeadersWriter(
+                        "Content-Security-Policy-Report-Only", "default-src 'self' fellesdatakatalog.digdir.no *.fellesdatakatalog.digdir.no"
+                    )
+                )
+                headers.addHeaderWriter(
+                    StaticHeadersWriter(
+                        "Reporting-Endpoints", "europe-west1-digdir-cloud-functions.cloudfunctions.net"
+                    )
+                )
+            }
+            .cors { cors -> cors.configurationSource { request -> CorsConfiguration().applyPermitDefaultValues() } }
```

Comment:
trenger ikke det variabelnavnet hvis du ikke skal bruke den
```suggestion
            .cors { cors -> cors.configurationSource { CorsConfiguration().applyPermitDefaultValues() } }
```